components/colorwheel/index.css

```diff
@@ -10,126 +10,128 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 .spectrum-ColorWheel {
-  --spectrum-colorwheel-width: var(--spectrum-color-wheel-width);
-  --spectrum-colorwheel-min-width: var(--spectrum-color-wheel-minimum-width);
-  --spectrum-colorwheel-height: var(--spectrum-color-wheel-width);
-  --spectrum-colorwheel-border-color: var(--spectrum-transparent-black-200);
-  --spectrum-colorwheel-border-width: var(--spectrum-border-width-100);
-  --spectrum-colorwheel-fill-color-disabled: var(--spectrum-disabled-background-color);
-  --spectrum-colorwheel-track-width: var(--spectrum-color-control-track-width);
-  --spectrum-colorwheel-colorarea-margin: var(--spectrum-color-wheel-color-area-margin);
-  --spectrum-colorwheel-colorhandle-position: calc((var(--spectrum-colorwheel-width) / 2) - (var(--spectrum-colorwheel-track-width) / 2));
+	--spectrum-colorwheel-width: var(--spectrum-color-wheel-width);
+	--spectrum-colorwheel-min-width: var(--spectrum-color-wheel-minimum-width);
+	--spectrum-colorwheel-height: var(--spectrum-color-wheel-width);
+	--spectrum-colorwheel-border-color-rgb: var(--spectrum-gray-1000-rgb);
+	--spectrum-colorwheel-border-opacity: 0.1;
```

Comment:
Is the `color-wheel-border-opacity` token undefined? Should we leave a note to Garth about that? Or a comment here about it being undefined?

Reply:
It is! I left him a note, but I'll add a todo as well. cc @GarthDB